Makefile

```diff
@@ -183,7 +183,7 @@ make-reports-dir:
 	mkdir -p $(REPORTS_DIR)
 
 test: make-reports-dir ## Run the unit tests
-	CGO_ENABLED=$(CGO_ENABLED) $(GOTEST) -count=1 $(COVERFLAGS) -failfast -short ./...
+	UNIT_TEST=true CGO_ENABLED=$(CGO_ENABLED) $(GOTEST) -count=1 $(COVERFLAGS) -failfast -short ./...
```

Comment:
This target isn't run during CI, you need to change this target https://github.com/jenkins-x/jx/blob/master/Makefile#L222